DESCRIPTION

ctf: Succeed at failing on OOME in tests

There is a test that fails with an OutOfMemoryError. This catches the
failure and thus makes the test succeed at failing.

Change-Id: If4b1a14bf135da3fae9f356631440acede56673c
Signed-off-by: Ivy Mitchell xxx@xxx.xxx
Signed-off-by: Braxton Mccarthy xxx@xxx.xxx


COMMENTS

author: Ivy Mitchell
date: 2013-12-02 23:08:56.000000000

Uploaded patch set 8.

-------------------------------------
author: Braxton Mccarthy
date: 2013-12-02 23:11:39.000000000

Patch Set 8: Code-Review+1 IP-Clean+1

It's still quite confusing :P

But it doesn't really matter, it's just a stopgap until we upgrade to Java 7.

-------------------------------------
author: Brenden Conley
date: 2013-12-03 00:32:39.000000000

Patch Set 8:

Build Started https://hudson.eclipse.org/linuxtools/job/linuxtools-gerrit/4349/

-------------------------------------
author: Brenden Conley
date: 2013-12-03 01:09:29.000000000

Patch Set 8: Verified+1

Build Successful 

https://hudson.eclipse.org/linuxtools/job/linuxtools-gerrit/4349/ : SUCCESS

-------------------------------------
author: Ivy Mitchell
date: 2013-12-03 01:59:15.000000000

Patch Set 8: Code-Review+2 IP-Clean+1

-------------------------------------
